python/ray/train/examples/mlflow_fashion_mnist_example.py

```diff
@@ -1,30 +1,26 @@
 import argparse
 
-import mlflow
-
 from ray.train import Trainer
 from ray.train.examples.train_fashion_mnist_example import train_func
+from ray.train.callbacks.logging import MLflowLoggerCallback
 
 
 def main(num_workers=2, use_gpu=False):
-    mlflow.set_experiment("train_torch_fashion_mnist")
-
     trainer = Trainer(
         backend="torch", num_workers=num_workers, use_gpu=use_gpu)
     trainer.start()
-    iterator = trainer.run_iterator(
+    final_results = trainer.run(
```

Comment:
Can you update the [examples](https://docs.ray.io/en/master/train/examples.html#iterator-api-examples) docs page for this? Currently this is labeled as an Iterator API example!

Reply:
Updated!